docs/usage_guide/examples/index.rst

```diff
@@ -0,0 +1,21 @@
+Modin Usage Examples
+====================
+
+This section shows Modin usage examples in different scenarios like Modin on a local/remote cluster,
+Modin in the cloud, the use of Modin spreadsheet and whatnot. First of all you may want to look at the basic
+Modin usage examples in tutorial_ directory. Then, you can check the use of Modin in different real benchmarks:
+
+* `NYC Taxi`_
+* Census
+* Plasticc
```

Comment:
No links available

Reply:
We do not have jupyter notebooks for Census and Plasticc.